components/input/ClearableLabeledInput.tsx

```diff
@@ -25,6 +25,7 @@ interface BasicProps {
   style?: object;
   disabled?: boolean;
   direction?: any;
+  focused?: boolean;
```

Comment:
Props?

Reply:
Yes.  This PR #19310 create an additional `ClearableLabeledInput` component for clearable but render input element is in `Input`. `focused` state is get from input and pass to `ClearableLabeledInput`.